### DISCUSSION TITLE
Anyone tried build performance masuring between NTFS and ReFS drives (dev drive)?

### DISCUSSION THREAD

**Discussion by xperiandri** [unoplatform/uno#18759] [Q&A]
I wonder if I get any performance improvement using ReFS dev drive

**Comment by jeromelaban** [comment]
It's supposed to improve performance, particularly on file copies between drivers (e.g. if your nuget folder is not on the ReFS drive, but your code is). Other parameters should improve perf, like filtering out your code folders from defender/anti-virus.

**Reply by MartinZikmund** [reply]
I feel there is a perf benefit, based on my usage. Don't have any specific numbers to prove it though.
